docs/headless-cms/03-02-cms-asyncapi-service.md

```diff
@@ -0,0 +1,12 @@
+---
+title: CMS AsyncAPI Service
+type: Details
+---
+
+CMS AsyncAPI Service is an HTTP server used in Kyma to process AsyncAPI specifications. It only accepts `multipart/form-data` forms and contains two endpoints:
+
+- `/validate` that validates the AsyncAPI specification against the AsyncAPI schema in version 2.0.0.
```

Comment:
You can mention that it uses https://github.com/asyncapi/parser :)

Reply:
Will do 👍